plugin/build.gradle

```diff
@@ -84,7 +84,7 @@ dependencies {
 	exclude group: 'com.google.errorprone', module: 'error_prone_annotations'
     }
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.11.0'
-    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.18.0'
```

Comment:
can we add this: https://github.com/opensearch-project/OpenSearch/blob/28b852d3a6d70a01ef64307db8ecdaa68becdcee/gradle/libs.versions.toml#L52

so that we don't have to keep maintaining versions by ourself. When core upgrades the version, we should be fine at the same time